docs/guides/random-data-for-security.md

```diff
@@ -137,6 +144,9 @@ We can also encode the string using [MIME::Base64](https://metacpan.org/pod/MIME
 There are times where you may want a more restricted alphabet, such as base-62. There are modules that let you generate
 random strings with custom alphabets or URL-safe encodings.
 
+Note that returning a message digest of random bytes adds no security.  Likewise, mixing random data with other
+information such as a timestamp or PID is unnecessary and does not improve the security.
```

Comment:
Indeed I suspect adding non-random data to randomness may make it less random...